disnake/stage_instance.py

```diff
@@ -152,6 +181,9 @@ async def edit(
         if privacy_level is not MISSING:
             if not isinstance(privacy_level, StagePrivacyLevel):
                 raise InvalidArgument("privacy_level field must be of type PrivacyLevel")
+            warn_deprecated(
```

Comment:
This should only warn when trying to set `privacy_level` to `StagePrivacyLevel.public`; the field itself isn't deprecated, just that specific value is, so there shouldn't be any deprecation warnings when accessing `privacy_level` either.